### ISSUE TITLE
Head and Footer

### ISSUE THREAD

**Issue by gumbey** [gumbey/bluevoice#2]
@chan0345 

Grand Goal
[-]Finish the header and footer
–––––––––––––––––––––––––––––––
[]-All links work
[]-Works on all screen sizes
[]-Follows Wireframes

**Comment by chan0345** [comment]
Goal almost complete:
• slogan to be added underneath logo
• change hover transition for Donate btn
All links work except for News and Films sublinks

**Comment by gumbey** [comment]
They won't work tell we get yaml to work

**Comment by chan0345** [comment]
@gumbey Yeah, I was just making a note that the links haven't been implemented yet

**Comment by elisak** [comment]
hey, the login on the footer needs to be moved! it's not aligned, also it's before the donate btn?
